<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sporeCameraClass</class>
 <widget class="QMainWindow" name="sporeCameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sporeCamera</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>计数结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>样品储藏孢子数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>个/g</string>
    </property>
   </widget>
   <widget class="QLabel" name="amount">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>99</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>37</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;正在识别图像。。。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>37</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;正在识别计数。。。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>该图中储藏孢子数：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="amount_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>个/g</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_6">
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>1000</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1000</height>
        </size>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>461</width>
          <height>451</height>
         </rect>
        </property>
        <widget class="QOpenGLWidget" name="openGLWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>0</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_5">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>0</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_6">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>0</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_7">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_8">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_9">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>70</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_11">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>70</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_12">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>140</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_13">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>140</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_14">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>140</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_15">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_16">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>140</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_17">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>210</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_18">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>210</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_19">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>210</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_20">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_21">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>210</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_22">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>280</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_23">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>280</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_24">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>280</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_25">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
        <widget class="QOpenGLWidget" name="openGLWidget_26">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>280</y>
           <width>81</width>
           <height>61</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>amount</zorder>
   <zorder>stackedWidget_2</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionsave"/>
    <addaction name="actionprint"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>计数</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionover"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>捕获</string>
    </property>
    <addaction name="actionsnap"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>识别</string>
    </property>
    <addaction name="actionstart_2"/>
    <addaction name="actionover_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>测量</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="actionhistory"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionsave"/>
   <addaction name="actionprint"/>
   <addaction name="actionsnap"/>
   <addaction name="actionhistory"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionsave">
   <property name="icon">
    <iconset>
     <normaloff>ico/save.png</normaloff>ico/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="icon">
    <iconset>
     <normaloff>ico/print.png</normaloff>ico/print.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>开始计数</string>
   </property>
  </action>
  <action name="actionover">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止计数</string>
   </property>
  </action>
  <action name="actionsnap">
   <property name="icon">
    <iconset>
     <normaloff>ico/snap.png</normaloff>ico/snap.png</iconset>
   </property>
   <property name="text">
    <string>创建快照</string>
   </property>
  </action>
  <action name="actionhistory">
   <property name="icon">
    <iconset>
     <normaloff>ico/picStore.png</normaloff>ico/picStore.png</iconset>
   </property>
   <property name="text">
    <string>图像库</string>
   </property>
  </action>
  <action name="actionstart_2">
   <property name="text">
    <string>开始识别</string>
   </property>
  </action>
  <action name="actionover_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止识别</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sporeCamera.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionstart_2</sender>
   <signal>triggered()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>menuScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>backMain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionhistory</sender>
   <signal>triggered()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>menuStore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionover_2</sender>
   <signal>triggered()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>menuScanOver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionstart</sender>
   <signal>triggered()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>menuCount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionover</sender>
   <signal>triggered()</signal>
   <receiver>sporeCameraClass</receiver>
   <slot>menuCountOver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuScan()</slot>
  <slot>menuStore()</slot>
  <slot>backMain()</slot>
  <slot>menuScanOver()</slot>
  <slot>menuCount()</slot>
  <slot>menuCountOver()</slot>
 </slots>
</ui>
